app/logic/daily_digest.rb

```diff
@@ -10,22 +10,25 @@ def perform
   private
 
   def get_statistic
-    recipients = get_recipients
-    completed = completed_amount
-    remaining = incompleted_amount
-
-    { recipients: recipients, completed: completed, remaining: remaining }
+    completed = completed_for_today
+
+    {
+      recipients: get_recipients,
+      completed_todos: completed.group_by { |item| item.list },
+      completed_amount: completed.size,
+      remaining_amount: incompleted
+    }
```

Comment:
@vlasiak It looks given changes are not reflected via tests. So there is no test that ensures proper data is obtained for the digest.